scripts/generator/graphql_generator/csharp/SDK/iOS/ShippingMethod.cs.erb

```diff
@@ -0,0 +1,20 @@
+#if UNITY_IPHONE
+namespace <%= namespace %>.SDK.iOS {
+    using System.Collections;
+    using System;
+
+    [Serializable]
+    public class ShippingMethod : SummaryItem {
+
+        #pragma warning disable 0414
```

Comment:
I think you might be able to kill these `warning disable` statements. You are using theser variabes in the constructor.